src/Smoker/SmokeTest.cs

```diff
@@ -150,10 +152,12 @@ public async Task<string> RunFromWebRequest()
                         // validate the response
                         if (resp.StatusCode == System.Net.HttpStatusCode.OK && r.Validation != null)
                         {
-                            res += ValidateContentType(r, resp);
+                            res = ValidateStatusCode(r, resp);
+                            res = ValidateContentType(r, resp);
                             res += ValidateContentLength(r, resp);
                             res += ValidateContains(r, body);
                             res += ValidateJsonArray(r, body);
```

Comment:
Add  res += ValidateExactMatch(r, body); here ?

Reply:
added ValidateAll() to resolve